apps/glusterfs/node_entry.go

```diff
@@ -159,13 +159,16 @@ func (n *NodeEntry) IsDeleteOk() bool {
 	return true
 }
 
+func (n *NodeEntry) ErrConflictFmt() error {
+	return fmt.Sprintf("Unable to delete node [%v] because it contains devices", n.Info.Id)
+}
+
 func (n *NodeEntry) Delete(tx *bolt.Tx) error {
 	godbc.Require(tx != nil)
 
 	// Check if the nodes still has drives
 	if !n.IsDeleteOk() {
-		logger.Warning("Unable to delete node [%v] because it contains devices", n.Info.Id)
-		return ErrConflict
+		return logger.Err(ErrConflict)
```

Comment:
Here, this is not an ERROR.  Same as the others.  You may want to have logger.Warning (and others) return `error`.  You can do that in another PR, and then have this one use that.